HiP-FeatureToggle/Controllers/FeatureGroupsController.cs

```diff
@@ -72,7 +72,7 @@ public IActionResult GetById(int groupId)
         [ProducesResponseType(422)]
```

Comment:
Add `[Authorize("write:featuretoggle")]` here, s.t. ppl without write privileges cannot call this method. Same for all other POST, PUT or DELETE calls in all the contorllers.